needle.pm

```diff
@@ -119,32 +119,12 @@ sub new ($classname, $jsonfile) {
     my $png = $self->{png} || $self->{name} . ".png";
 
     $self->{png} = path(dirname($jsonfile), $png)->to_string;
-    warn "Can't find $self->{png}" and return unless -s $self->{png};
+    warn "Can't find $self->{png}" and return unless -e $self->{png};
```

Comment:
```
    -e: Checks if the file exists.
    -s: Checks if the file is non-empty.
```

Since the warning is for file not found I've updated with a check of the existing file

Reply:
That changes the behaviour, and maybe it is good to check for a non-empty file though. Maybe the warning should be updated to reflect that instead.

Reply:
Using -s will break the handle tags duplicates, but I'll populate the tag.png with random string!